src/components/tabs/buy-tab-content/buy.overview.tsx

```diff
@@ -19,16 +21,18 @@ export function BuyTabContentOverview({ onAssetClicked }: BuyTabContentOverviewP
     <StyledVerticalStack gap={0}>
       {Array.from(assets.entries()).map(([blockchain, assets], blockchainIndex) => (
         <StyledCoinList key={blockchainIndex} heading={toHeader(blockchain)}>
-          {assets.map((asset, assetIndex) => (
-            <StyledCoinListItem
-              key={assetIndex}
-              asset={asset}
-              protocol={toProtocol(blockchain)}
-              onClick={() => onAssetClicked(asset)}
-              popupLabel="Click on the MetaMask symbol in order to add this asset in your portfolio overview of your MetaMask or copy the address to add it manually."
-              onAdd={addContract}
-            />
-          ))}
+          {assets
+            .filter((a) => a.buyable || a.comingSoon)
+            .map((asset, assetIndex) => (
+              <StyledCoinListItem
+                key={assetIndex}
```

Comment:
AFAIK it's generally not recommended to use the array index as key, because the list may change. I think in this case it doesn't really matter though.

https://react.dev/learn/rendering-lists#why-does-react-need-keys

_You might be tempted to use an item’s index in the array as its key. In fact, that’s what React will use if you don’t specify a key at all. But the order in which you render items will change over time if an item is inserted, deleted, or if the array gets reordered. Index as a key often leads to subtle and confusing bugs._